.gitignore

```diff
@@ -32,3 +32,5 @@ custom-claim-handler/target/
 custom-permission-claim-handler/target/
 custom-password-generator/target/
 custom-user-operation-event-listener/target/
+custom-remote-user-store-management/agent/target
```

Comment:
custom-remote-user-store-manager/agent/target

Reply:
why add the targets one by one. add just below line
target

Reply:
Refactored in a7f6d20d13ff0c022c209a6315a7b2c9fab132ce